### ISSUE TITLE
FLACCL Exception: CreateContext failed with error code: 222

### ISSUE THREAD

**Issue by kamenminkov** [gchudov/cuetools.net#206]
I'm getting the error in the title every time I try to use FLACCL. Encoder settings are left to default. It doesn't seem to matter which mode (0-8) I'm using.

GPU is an RTX 3070 running driver version 31.0.15.2514 (525.14), OS is Win 11 Dev (25158.1000).

Please let me know what other info could be useful. Sorry if this is a FLACCL issue and not a CUETools issue.

**Comment by c72578** [comment]
@kamenminkov Thanks for your report.
Could you please also post the CUETools version, which you are using.
Have you already tried a different NVIDIA driver version? There has been a report recently about working Nvidia RTX 3080 and Nvidia RTX 4090 graphics cards (#249). In that case, driver version `528.02` was installed and `CUETools.FLACCL.cmd.exe` used for encoding.

**Comment by kamenminkov** [comment]
@c72578 Thanks for pinging me. I had CUETools switched to the cuetools encoder and hadn't tried FLACCL in a while. Just tried FLACCL again and this time it actually seems fine (tried two different files and both work). CUETools is now `2.2.3` and the driver version is `528.24`. Windows is a newer build, `25284.1000`. Hardware is the same.

I _think_ I was using CUETools `2.2.2` when I wrote this originally.

I just got curious and installed `2.2.2` again - and it's working fine now too - so at least I think that this most likely points to a driver issue.
